pkg/replication/service.go

```diff
@@ -258,6 +265,15 @@ func (s *svc) getReplicationPolicy(ctx context.Context, task tasks.SyncTask) (po
 	if !errors.Is(err, dom.ErrNotFound) {
 		return policy.ReplicationPolicies{}, err
 	}
+	// if zero-downtime switch is in progress return empty replication policy
+	// to update obj version metadata and avoid creating replication tasks
+	_, err = s.policySvc.GetInProgressZeroDowntimeSwitchInfo(ctx, user, bucket)
+	if err == nil {
```

Comment:
Should it be `err == nil`?

Reply:
yes, it is correct. added comment to code